index.bs

```diff
@@ -1838,6 +1839,18 @@ a [=header value=] or null |webTriggerHeader|, a [=header value=] or null
 
      </dl>
 
+To <dfn export>get the first non-coalesced header</dfn> given a [=response/header list=] |headers|,
+and a [=header name=] header:
+1. Let |non-coalesced-headers| be the result of [=get non-coalesced headers=] with |headers| and |header|.
+1. If |non-coalesced-headers| [=list/is empty=], return null.
+1. Return |non-coalesced-headers|[0].
+
+To <dfn export>get non-coalesced headers</dfn> given a [=response/header list=] |headers|,
+and a [=header name=] header:
+1. If |headers| does not [=header list/contain=] |header|, return a new [=list=].
+1. Return the [=header value|values=] of all [=header|headers=] in |headers| whose [=header name|name=]
+    is a [=byte case insensitive=] match for |header|, in order.
```

Comment:
```suggestion
    is a [=byte-case-insensitive=] match for |header|, in order.
```

Reply:
It is defined at the top with "byte case insensitive".

Reply:
Right, but that's not how it's spelled in the fetch spec: https://infra.spec.whatwg.org/#byte-case-insensitive

Reply:
I removed the mapping and use the link directly instead.